app/javascript/services/fetchAudits.jsx

```diff
@@ -0,0 +1,30 @@
+import apiRequest from "./api/apiRequest";
+
+const fetchAudits = async (filters) => {
+  /**
+   * Build the string filter to pass to the HTTP GET operation
+   *
+   * @param {Object} filters
+   */
+  const buildFilter = (filters) => {
+    let filterString =
```

Comment:
I'd suggest using `query-string` for query params building instead of doing it manually.